jobs/config.json

```diff
@@ -7126,7 +7126,7 @@
       "--gke-command-group=beta",
       "--gke-environment=test",
       "--provider=gke",
-      "--test_args=--gce-multizone=true --ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\] --minStartupPods=8",
+      "--test_args=--gce-multizone=true --gce-region=us-central1 --ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\] --minStartupPods=8",
```

Comment:
Isn't it gcp-region?

Reply:
No.
--gcp-region is a flag to kubetest (that is not propagated to test binary)

--gce-region is the flag of e2e binary

Reply:
Aah.. right. It's part of test-args.
That was confusing.